src/main/java/htsjdk/samtools/cram/encoding/reader/CramRecordReader.java

```diff
@@ -326,7 +327,7 @@ public void read(final CramCompressionRecord cramRecord) {
                 System.err.printf("Failed at record %d. Here is the previously read record: %s\n", recordCounter,
                         prevRecord.toString());
             }
-            throw new RuntimeException(e);
+            throw new RuntimeIOException(e);
```

Comment:
@jmthibault79 This one isn't really an IO exception. In a future pass, we just should completely remove the entire try/catch block in this method if possible - it looks like its just temporary code for debugging purposes, and writes to stderr instead of logging, etc.